### ISSUE TITLE
[SC and HR]: Multiple errors triggered at once on a page aren't all announced

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#103724]
## What happened?

**Describe what went wrong**:

If multiple errors are triggered on a page at once, only one error gets announced by some screen readers.

This isn't a show stopping bug though, but one where the  experience can be improved. It's not a show stopper because:

1. The inputs with the errors will announce as invalid, per design system component code, helping people who use screen readers know that there's an issue.
2. In the case of our forms, we move focus to the top most error, which will help people find the other error(s) as they move forward in the forms.

However, it's challenging because:

1. People who use screen readers may not know that multiple errors occurred without additional exploration of a page.
2. It's harder to explore the page by tabbing through the inputs without filling them out as a way to understand what's expected because errors will announce as a person leaves the input.

**Add the steps to reproduce the bug**:

1. Log into Staging and visit the Supplemental Claims form or Higher Level Review form.
2. Go to a page that has multiple required fields. For the purposes of my testing, I used the Supplmmental Claims form and the "Add a first VA or military treatment location" page. It's here: `https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/va-medical-records`
3. Turn on a screen reader, for example in Safari with VoiceOver.
4. Navigate through the fields and reach the continue button.
5. Activate the continue button.
6. Note how focus moves to the top most error. Only one error gets announced.

**Add a screen recording or photo**:

Experience in Safari on VoiceOver on Mac:

https://github.com/user-attachments/assets/45ee714f-8bd9-4dd2-a15b-f42ffb4cbda9

Experience in Firefox on NVDA on Windows:

https://github.com/user-attachments/assets/d02a6939-3d86-40f5-be07-4194cf9f7fdb

Experience in Chrome on JAWS on Windows:

https://github.com/user-attachments/assets/35b05c23-3832-4214-9297-b649653f4a98

Some details:

- Safari/VoiceOver announced the error focused on and not both.
- Firefox/NVDA announced both errors.
- Chrome/JAWS announced the error focused on and not both.

**Possible solution**

- Create an error summary at the top of the page when more than one error occurs at the same time.
- A [design exploration and specs are in a Figma file](https://www.figma.com/design/sMk1jDYRpFRY4buTX1kHhk/Error-Exploration?node-id=0-1&p=f&t=KAdNsS18UVno9pG9-0).
- I'm also going to file an experimental design ticket with the design system.
- cc: @eileen-coforma and @Mottie for reference.

## How severe is the problem?

- [ ] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [x] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user does X, the **visual** result is y._
2. _When a user does X, the **backend** result is y._
3. _When a user does X, the **event logging** result is y._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place
